database/postgresql_adapter.go

```diff
@@ -155,8 +160,13 @@ func (a *postgreSqlAdapter) copyHeaders(reader *csv.Reader, batchSize int, previ
 		if len(record) == 0 {
 			break
 		}
+		var b dto.DbBlockHeader
+		b, err = PrepareRecord(record, lastBlockHash, bh, cumulatedChainWork, lastRowIndex)
+		if err != nil {
+			fmt.Printf("Error while preparing record: %v", err.Error())
+			os.Exit(1)
```

Comment:
I think we shouldn't Exit here, because BHS can always get headers from peers from even the very beginning if the import fails. But maybe it's a topic for discussion on the channel

Reply:
IMO we can just return error here.

Reply:
changed to return error